_posts/2016-01-13-Get-Inspired-on-GitHub.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Get Inspired on GitHub
+excerpt: Learn how to search for API design patterns on GitHub.
+layout: post
+date: 2016-01-13 00:00:00 +0100
+author: zdenek
+published: true
+comments: true
+---
+
+Last June API Blueprint became recognized as a language on GitHub.
+This brought the syntax highlighting and repository statistics for API
+Blueprint. More importantly, it has also enabled the search for API design
+patterns.
+
+Let’s take a quick look on how we can search and perhaps learn from the API
+designs done by others.
+
+# Search
+
+To search for API Blueprint on GitHub simply go to
+
+[https://github.com/search](https://github.com/search)
+
+<img width="640" src="/images/2016-01-13-Get-Inspired-on-GitHub/search.png" alt="search" />
+
+And enter the `language:"API Blueprint"` as the query. Hitting the search will
+list all the GitHub repositories that contains at least one API Blueprint
+(.apib) file.
+
+<img width="640" src="/images/2016-01-13-Get-Inspired-on-GitHub/statistics.png" alt="statistics" />
+
+Seeing every blueprint on GitHub could be fun but what is more fun is to search
+for a particular keyword in blueprints. Here are all the blueprints that
+contain the infamous "Hello World":
+
+<img width="640" src="/images/2016-01-13-Get-Inspired-on-GitHub/hello.png" alt="hello" />
+
+Note: When you are searching for query like "Hello World" make sure to switch to
+ the "Code" results in the left hand pane.
+
+# Patterns
+
+Ok, it is the time to look how others are solving some popular API design
+patterns.
+
+## Pagination
+
+Is your resource a collection of elements or other resources, such as a result
+of search? Maybe it is a good idea to split the response into chunks—pages.
+
+Searching blueprints for the `page` word seems like to be a good start to see
```

Comment:
It looks like you are switching between quotes and backticks for keywords for the search. It would be good to have these the same.